### ISSUE TITLE
Process exited with code 132

### ISSUE THREAD

**Issue by mxcoldhit** [OpenDroneMap/WebODM#330]
How did you install WebODM? (Docker, natively, ...)?
Docker

What's your browser and operating system? (Copy/paste the output of https://www.whatismybrowser.com/)
Chrome 61 on Windows 10

What is the problem?
I get the following error after starting my first task using the web interface. Using 45 images
/code/run.sh: line 6:    24 Illegal instruction     python $RUNPATH/run.py "$@"

What should be the expected behavior?

How can we reproduce this? (What steps did you do to trigger the problem? What parameters are you using for processing? If possible please include a copy of your dataset uploaded on Google Drive or Dropbox. Be detailed)
I am using the default options. Just tried to process my first set if images.

**Comment by pierotofy** [comment]
Hi @mxcoldhit,

It looks like there is some information missing from your ticket that will be needed in order to diagnose and fix the problem at hand. Please take a look at the [Issue Template](https://github.com/OpenDroneMap/WebODM/blob/master/ISSUE_TEMPLATE.md), which will tell you **exactly** what your ticket has to contain in order to be processable.

I'm marking this one now as needing some more information. Please understand that if you do not provide that information within the next week (until 2017-11-23 15:50) I'll close this ticket so it doesn't clutter the bug tracker.

Best regards,
~ Your friendly GitIssueBot

PS: I'm just an automated script, not a human being.

**Comment by pierotofy** [comment]
Hi @mxcoldhit, you are probably using an older CPU. The docker setup needs CPUs with 64-bit extensions, MMX, SSE, SSE2, SSE3 and SSSE3 instruction set support or higher.

You could rebuild the OpenDroneMap and node-OpenDroneMap docker images from sources and then use that instance of node-OpenDroneMap to process a task.

**Comment by pierotofy** [comment]
Hey @julianpalladino :hand: , no, that will only rebuild the WebODM containers.

You need to rebuild ODM (https://github.com/OpenDroneMap/OpenDroneMap), then node-ODM (https://github.com/OpenDroneMap/node-OpenDroneMap), then finally you can use the node-ODM image with WebODM (no need to recompile WebODM's containers). Usually a:

`docker build -t <appropriate tag name> .`

Will do the trick for each repository.

**Comment by julianpalladino** [comment]
Hi again @pierotofy  , thank you very much for your help!
I tried building OpenDroneMap and node-OpenDroneMap natively. However I still have the same problem (code 132 / illegal instruction, due to old CPU), exactly as [this issue](https://github.com/OpenDroneMap/OpenDroneMap/issues/859).

Is it a bug? Is there a way to bypass that part of the program? I wouldn't mind to lose some functionality, as long as I can use the rest of the program

**Comment by pierotofy** [comment]
Hey @julianpalladino this is a great find! Would you mind sharing more about the process you followed to remove the flag? It would be great to document the fix for other users (or even open a PR).

**Comment by fragofer** [comment]
Hi @pierotofy, we're working with @julianpalladino on this issue. 
The problem wasn't in ODM, node-ODM nor WebODM so rebuilding these projects led us to the same error. After long hours of debugging we found that the problem was/is in a related project MVE (used by mvs-texturing): https://github.com/nmoehrle/mve 

In particular, in the file:
https://github.com/simonfuhrmann/mve/blob/master/Makefile.inc
They added the flag -sse4 which is not supported by old AMD processors (ours is an AMD Phenom II X6 1090T). So, we removed that flag and now everything is working :-)

I think that the best thing to do is to transform MVE in a CMakeList based project, so for now a good fix would be to change:
https://github.com/OpenDroneMap/mvs-texturing/blob/master/elibs/CMakeLists.txt
and point to forked projects that can be controlled by the ODM team, and also try to avoid cloning external libraries from master branches to gain stability in the building process.

Thanks!
Francisco
